_translations/en.yml

```diff
@@ -185,6 +185,7 @@ en:
     walletcatweb: "Web"
     walletbitcoincore: "Bitcoin Core is a full Bitcoin client and builds the backbone of the network. It offers high levels of security, privacy, and stability. However, it has fewer features and it takes a lot of space and memory."
     walletbitcoinknots: "Bitcoin Knots is a full Bitcoin client and builds the backbone of the network. It offers high levels of security, privacy, and stability. It includes more advanced features than Bitcoin Core, but they are not as well-tested. It uses a lot of space and memory."
+    walletcobo: "Cobo is a company that is firmly committed to supporting blockchain development and making cryptocurrencies easy to own and use. We emphasize long-term security, reliability, and convenience. Cobo Wallet is our flagship mobile wallet. It’s also the first wallet in the world to offer pooled staking, masternode rewards, and currently supports more than 30 mainchains and 500 tokens."
```

Comment:
The description should be no more than 320 characters.

Reply:
The description itself also seems more targeted to tokens and makes no mention of bitcoin.

Reply:
Also, the description starts out talking about the company.  It definitely should describe the wallet instead.  Please look at the other wallet descriptions as examples.

Reply:
Thank you guys! I will change it.